setup.py

```diff
@@ -267,6 +267,7 @@ def run(self):
         version=version,
         install_requires=fetch_requirements(),
         packages=setuptools.find_packages(exclude=("tests", "tests.*")),
+        dependency_links=["file:///./third_party/flash-attention#egg=flash-attention"],
```

Comment:
@danthe3rd let's test this afterwards. I think we will need to change the way we depend on flashattention in our code